data_generator.py

```diff
@@ -0,0 +1,36 @@
+import random
+
+first_names = ["James", "Mary", "Liam", "Olivia", "Robert", "Patricia", "Noah", "Emma", "John", "Jennifer"]
+middle_names = ["Louise", "Rose", "Grace", "William", "Jane", "Elizabeth", "Thomas", "Anne", "Ann", "David"]
+last_names = ["Smith", "Johnson", "Williams", "Brown", "Jones", "Garcia", "Miller", "Davis", "Rodriguez", "Martinez"]
+
+names = []
+
+for first_name in first_names:
+    for last_name in last_names:
+        names.append(first_name + ' ' + last_name)
+
+for first_name in first_names:
+    for middle_name in middle_names:
+        for last_name in last_names:
+            names.append(first_name + ' ' + middle_name + ' ' + last_name)
+
+countries = ["Switzerland", "Canada", "Sweden", "Australia", "United States", "Japan", "Germany", "New Zealand"]
+
+vaccines = [["Pfizer", "Mainz, Germany"], ["Moderna", "Massachusetts, U.S."], ["Novavax", "Maryland, U.S."]]
+
+with open("data_entry.tql", 'w') as f:
+
+    for curr_vaccine in vaccines:

```

Comment:
There's no need for the `curr_` prefix on this variable name. The fact that it's the one you're currently operating on is implicit by the use of an iterator.